scripts/patrol-gui.lua

```diff
@@ -672,6 +699,9 @@ function PatrolGuiWaypoint.waypoint_type_changed(player, spidertron, gui_element
       waypoint.wait_time = 5
     elseif new_waypoint_type == "submerge" then
       waypoint.wait_time = 2
+    else
```

Comment:
This should not be here.

Reply:
As far as I can see, this is a part of the code that lets the entity continue moving after I teleported it.

Reply:
namely this bit.
https://github.com/Frontrider/SpidertronPatrols/blob/master/scripts/spidertron-control.lua#L147-L150

Reply:
specifically the `else` is almost certainly a syntax error (have you tried running this file)

Reply:
I will admit I am working from 2 folders right now.

Reply:
I'd strongly suggest at least getting a portable zip version of 2.0.39 then, as being able to run the code your submitting seems pretty critical :)

Reply:
My answer to "trying to run it".
[liftoff.webm](https://github.com/user-attachments/assets/90882711-1514-4f4e-a4ae-d083a0df72fa)

Reply:
I am running it now, on the beta, something is crashing.
I need to take a break from it, I'll check tomorrow.